pkg/uibackend/api/openapi.yaml

```diff
@@ -4,9 +4,6 @@ info:
   title: VMClarity UI APIs
   version: 1.0.0
 
-servers:
-  - url: /ui/api
-
```

Comment:
why it was removed?

Reply:
Same reason as the apiserver, the microservice doesn't serve the API on a subpath because its a dedicated endpoint now. The API gateway handles the /ui/api subpath.